modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/com/liferay/dynamic/data/mapping/dependencies/ddm/color.ftl

```diff
@@ -7,29 +7,28 @@
 <@liferay_aui["field-wrapper"]
 	cssClass="form-builder-field"
 	data=data
+	label=escape(label)
 	required=required
 >
 	<div class="form-group">
 		<@liferay_aui.input
 			cssClass="selector-input"
 			helpMessage=escape(fieldStructure.tip)
 			inlineField=true
-			label=escape(label)
+			label=""
 			name="${namespacedFieldName}Color"
 			readonly="readonly"
+			title=escape(label)
 			type="text"
 			value=escape(fieldStructure.tip)
 		/>
 
 		<@liferay_aui.input
 			cssClass="color-value"
 			name=namespacedFieldName
+			required=required
```

Comment:
Do we really want to make the hidden input have the required validation?

Reply:
It needs to be in the hidden input since the input seems to only be for display and will not work correctly.